.travis.yml

```
@@ -1,8 +1,18 @@
 language: node_js
-node_js:
-- '12'
-- '10'
-dist: bionic
+matrix:
+  include:
+    - os: linux
+      dist: bionic
+      node_js: '10'
+    - os: linux
+      dist: bionic
+      node_js: '12'
+    - os: osx
+      osx_image: xcode12
+      node_js: '10'
+    - os: osx
+      osx_image: xcode12
+      node_js: '12'
```

Not sure why this is in a matrix.

```suggestion
os:
- linux
- osx
node_js:
- '10'
- '12'
dist: bionic
osx_image: xcode12
```

---

You are right! It is my first time with travis configuration and I've been a bit lost 😅
I just applied your suggestion 👍